__tests__/app.test.js

```diff
@@ -193,6 +193,72 @@ describe("/api/articles/:article_id/comments", () => {
             });
         });
     });
+    describe('POST', ()  => {
+        it('should respond with 201 and add comment for specified article and respond with the posted comment', ()    => {
+            return request(app)
+            .post("/api/articles/7/comments")
+            .send({
+                   username: "lurker",
+                   body: "Hellllloooooo world!"
+                  })
+            .expect(201)
+            .then((response) => {
+                const comment = response.body.addedComment;
+                const convertTime =  convertTimestampToDate(comment.created_at);
+                const date = (Object.values(convertTime).join(''))
+                const currentTime = new Date()
+                expect(Object.keys(comment).length).toBe(6);
+                expect(comment).toMatchObject({
+                    article_id: 7,
+                    author: "lurker",
+                    body: "Hellllloooooo world!",
+                    comment_id: 19,
+                    created_at: date,
+                    votes: 0  
+                });
+        });
+        });
+        it('status code 400 with appropriate message when key is missing from input body', ()    =>   {
+        return request(app)
+        .post("/api/articles/7/comments")
+        .send({
+                body: "bodyodyodyody"
+             })
+        .expect(400)
+        .then((response) =>  {
+                expect(response.body.msg).toBe('Bad request')
+            });
+        });
+        it('status code 404 when invalid value inputted', ()    =>   {
+        return request(app)
+        .post("/api/articles/7/comments")
+        .send({
+            username: "HP",
+            body: "Hellllloooooo world!"
+           })
+        .expect(404)
+        .then((response) =>  {
+            expect(response.body.msg).toBe('Recieved invalid value')
+        });
+    });
+    it('404: should respond with appropriate message when article_id is valid but outlying any stored ids', ()   =>  {
+        return request(app)
+        .get("/api/articles/500/comments")
```

Comment:
I think these next two tests should be .post, not .get? Be careful if we've copy and pasted here, these tests might have auto passed as it's using functionality from another endpoint, if it did then it's usually a good indicator that we have made a mistake writing the test.